sklearn/metrics/_plot/regression.py

```diff
@@ -0,0 +1,387 @@
+import numbers
+
+import numpy as np
+
+from ...utils import check_matplotlib_support
+from ...utils import check_random_state
+from ...utils import _safe_indexing
+
+
+class PredictionErrorDisplay:
+    """Prediction error visualization.
+
+    It is recommended to use
+    :func:`~sklearn.metrics.PredictionErrorDisplay.from_estimator` or
+    :func:`~sklearn.metrics.PredictionErrorDisplay.from_predictions` to
+    create a visualizer. All parameters are stored as attributes.
+
+    Read more in the :ref:`User Guide <visualizations>`.
```

Comment:
we should also refer to the relevant section of the "model selection" chapter here.